### PR TITLE
[doc] Delete duplicate page contributing.md on the website

### PR DESCRIPTION
**Pull request by ishanSrt**
Closes https://github.com/pmd/pmd/issues/994
Please, prefix the PR title with the language it applies to within brackets, such as *[java]* or *[apex]*. If not specific to a language, you can use *[core]*

Before submitting a PR, please check that:
 - [x] The PR is submitted against `master`. The PMD team will merge back to support branches as needed.
 - [x] `mvn test` passes.
 - [x] `mvn checkstyle:check` passes. [Check this for more info](https://github.com/pmd/pmd/blob/master/CONTRIBUTING.md#code-style)

**PR Description:**

### CODE REVIEW

File: docs/pages/pmd/devdocs/contributing.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
```
Comment (adangel) [inline docs/pages/pmd/devdocs/contributing.md]: I think, we can delete this whole file. I didn't see any other internal link to this page except for the sidebar (which you already adjusted).

**Comment by ishanSrt** [comment]
@adangel I did what you said but I don't understand what am I doing
 Here's what I can make out:
* You guys build the entire site using some prewritten script using the docs directory for each release
* So when I am deleting contributing.md this will prevent the page `contributing` to be built on to this site and I am not deleting the actual `contributing.md` which seems to be located directly inside the pdb directory

correct me if I am wrong

**Comment by adangel** [comment]
The file [pmd/docs/devdocs/contributing](https://github.com/pmd/pmd/blob/master/docs/pages/pmd/devdocs/contributing.md) is transformed by [Jekyll](https://jekyllrb.com/) into e.g. https://pmd.github.io/pmd/pmd_devdocs_contributing.html . We take for each release a snapshot on the configuration and make it available under a fixed url with the version number in it, e.g. https://pmd.github.io/pmd-6.2.0/pmd_devdocs_contributing.html

The other file [CONTRIBUTING.md](https://github.com/pmd/pmd/blob/master/CONTRIBUTING.md) (note uppercase and other directory outside of `docs`) is unrelated in terms of documentation building. With your change, we just link PMD's source code on github.

**Comment by ishanSrt** [comment]
Thank **you** for guiding me!

**Comment by adangel** [comment]
Merged! Congrats to your first PR! :tada:

**Pull request merged**